app/Repositories/Eloquents/TopicEloquentRepository.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/2/18
+ * Time: 1:00 PM
+ */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/TopicEloquentRepository.php</br>**Line**: 9</br>There must be one blank line after the namespace declaration